star_distribution.py

```diff
@@ -0,0 +1,119 @@
+import math
+import numpy as np
+import matplotlib.pyplot as plt
+
+# define the rotation curve
+def vel(r1):
+
+    ar1 = math.fabs(r1)
+    if (ar1 <= 5.0):
+        v1 = 1.4*ar1*math.exp(-ar1/20.0)
```

Comment:
All of this math could be explained, either with line-by-line comments or by explaining the equations you're using in the function docstring